### PR TITLE
Missing metadata and files alert

### PR DESCRIPTION
**Pull request by pkdash**
This alert now disappears as soon as the requirements are met. No page reload is needed.

### CODE REVIEW

**Comment by pkdash** [comment]
@hyi or @mauriel133 Can one of you review this small change?

File: hs_core/views/resource_folder_hierarchy.py [hunk 1]
```diff
@@ -72,8 +72,10 @@ def data_store_structure(request):
     except SessionException as ex:
         return HttpResponse(ex.stderr, status=500)
 
+    can_be_public = "true" if resource.can_be_public_or_discoverable else 'false'
```
Comment (hyi) [inline hs_core/views/resource_folder_hierarchy.py]: @pkdash This can be simplified as ```can_be_public=resource.can_be_public_or_discoverable```

**Comment by hyi** [comment]
@pkdash Just one minor comment to be addressed. Then a +1 from me.

**Comment by pkdash** [comment]
@hyi Once Jenkins checks are successful I will be merging this.

**Comment by pkdash** [comment]
@hyi When I made this change, in debug mode I saw JS gets the value for ```can_be_public``` as a Boolean value and not a string. So it works as implemented.

**Comment by hyi** [comment]
@pkdash OK. That is good to know. Then it should be good.

**Pull request merged**